tests/unit/mock_data/counts/test_workflow_status_succeded_workflow_slim_metadata.txt

```diff
@@ -0,0 +1,8 @@
+	ScatterAt51_18	0 Running, 1 Done, 0 Preempted, 0 Failed
```

Comment:
Why the leading whitespaces?

Reply:
When running counts, some workflow info is first printed then the status of the task for that workflow is printed after a tab. The test asserting the function is expecting tabs

```
workflow name
workflow id
        task 0 Running, 1 Done, 0 Preempted, 0 Failed
        task 0 Running, 1 Done, 0 Preempted, 0 Failed
```